### ISSUE TITLE
Local development mode works but generated page is broken after update

### ISSUE THREAD

**Issue by ottlinger** [zjedi/hugo-scroll#123]
I've updated my page to the newest version of the plugin
https://www.aikido-entspannung.de/

Local development works fine, but the generated page seems to be missing the stuff around .... :(

I've tried reinitialising the project and submodule with no success.
Config.toml is basically a copy of the one from exampleSite. The strange thing is that local development via:
`hugo server
`
works fine and shows the page as expected with red background.

If I generate the page for deployment via:
`hugo --gc --minify --enableGitInfo
`
and upload it to my host the page is broken.

Do you have any hints on how to fix this?
(Page worked perfectly well with the last version from Jan before)

Thanks

**Comment by zjedi** [comment]
Can you try running without the extra commandline  arguments gc, minify, enableGitInfo if that helps and add them one by one? To see which one is causing issues

The live demo seems alive (automatically updated from master) so it must be one of these arguments.

**Comment by zjedi** [comment]
When I open your site with dev tools, it looks like the CSS is completely missing. Do you have any customizations aroud it?

**Comment by ottlinger** [comment]
Thanks for your hints - I did some local mix of
`npm install` and `npm update` locally after setting up the project once more.
I'll have a look at it when I do more changes in contents.

**Comment by ottlinger** [comment]
The problem was related to the baseURL. I configured my live-URL in config.toml.
Changing my dev scripts to `--baseURL .` solved the problem locally.

**Comment by ottlinger** [comment]
@zjedi do you have any idea about what might be wrong: my page shows fine on desktops but using a firefox klar on an android device still renders the homepage without css?!
https://www.aikido-entspannung.de/

I tried the demo of the theme, which works fine, so it must be some change that applies or a rule that prevents the browser from loading/applying the css ..... a bit clueless, therefore just a comment here .... 

thanks

**Comment by ottlinger** [comment]
The problem seems to be related to a difference between 
* https://www.aikido-entspannung.de/ - renders the page without design
* https://aikido-entspannung.de/ - shows properly
not sure what to do/configure to properly support multiple hosts.

**Comment by zjedi** [comment]
> The problem seems to be related to a difference between
> 
> * https://www.aikido-entspannung.de/ - renders the page without design
> * https://aikido-entspannung.de/ - shows properly and is the configured baseURL
>   not sure what to do/configure to properly support multiple hosts.

I don't think you can configure multiple hosts. In your hosting it's the same directory, right?
I think you just need to setup automatic redirection from one url form to another, at your hosting provider

**Comment by ottlinger** [comment]
@zjedi thanks, I reconfigured the webserver. In order to avoid redirects I configured multiple webservers for various URIs. Now I have a redirect to www, which is configured as baseURL. Hopefully this will work out.
